gadgets/src/fields/fp12.rs

```diff
@@ -672,15 +672,15 @@ where
     <P::Fp6Params as Fp6Parameters>::Fp2Params: Fp2Parameters<Fp = F>,
 {
     fn to_bits_be<CS: ConstraintSystem<F>>(&self, mut cs: CS) -> Result<Vec<Boolean>, SynthesisError> {
-        let mut c0 = self.c0.to_bits_be(&mut cs)?;
-        let mut c1 = self.c1.to_bits_be(cs)?;
+        let mut c0 = self.c0.to_bits_be(cs.ns(|| "c0"))?;
+        let mut c1 = self.c1.to_bits_be(cs.ns(|| "c1"))?;
```

Comment:
This fixes an existing issue about `to_bits_le/be` for Fp2/Fp6/Fp12.

Before this change, it would cause the constraint system to complain that the path has been used.